app/models/issue.rb

```diff
@@ -19,11 +19,18 @@ def get_interfaces(interface = nil)
   end
 
   def get_headers(header = nil)
-    all_headers = get_interfaces(:http)._data[:headers]
+    all_headers = http_data(:headers)
     return all_headers if header.nil?
     all_headers.find { |h| h[header] }.try(:values).try(:first)
   end
 
+  def http_data(key = nil)
+    all_data = get_interfaces(:http)._data
+    return '<no information>' if all_data.blank?
```

Comment:
In the image you gave me, I remember there was a <no information> message in the error occurred email in the Url section.

This was my approach 
%p.gray_text Url
    %pre
      %code
        %small #{@issue.http_data(:url)}